proto/agglayer/interop/types/v1/aggchain.proto

```diff
@@ -8,10 +8,40 @@ import "agglayer/interop/types/v1/bytes.proto";
 message AggchainData {
   // The data of the aggchain.
   oneof data {
-    // Signature committed to the bridge exits and imported bridge exits.
+    // Legacy ECDSA signature on a commitment to the state transition.
     FixedBytes65 signature = 1;
-    // Generic Aggchain proof.
+
+    // Aggchain proof with single ECDSA verified by the agglayer only.
     AggchainProof generic = 2;
+
+    // Multisig only performed by the chain's committee.
```

Comment:
```suggestion
    // Multisig performed by the chain's committee on a commitment to the network state transition.
```

Reply:
given how much discussions happened on the commitment, I feel better to _not sprinkle hints on what is signed everywhere in the codebase, I think it's better to isolate any doc+explanations about this where the commitment is actually defined (on the agglayer side)